### PR TITLE
feat: add dash folder and deploy example

### PR DESCRIPTION
**Pull request by jravenel**
adding dash deployment example

### CODE REVIEW

**Comment by Dr0p42** [comment]
No linked issues found. Please add the corresponding issues in the pull request description. <br/> [Use GitHub automation to close the issue when a PR is merged](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

**Comment by jravenel** [comment]
@Dr0p42 I don't know why it's blocking I face this issue already, it says we do not have Issue linked but there is. 
I think we should simply remove this and let people do direct PR or generate Issue automatically? Don't you think?

**Comment by jravenel** [comment]
This PR resolve: https://github.com/jupyter-naas/awesome-notebooks/issues/1127

**Comment by jravenel** [comment]
@eliasdabbas all my checks have passed, can you review?

File: Dash/Dash_Deploy_app_in_Naas.ipynb [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+     "shell.execute_reply": "2022-07-31T19:05:32.588289Z",
+     "shell.execute_reply.started": "2022-07-31T19:05:32.549012Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "df = px.data.stocks() #reading stock price dataset \n",
+    "def stock_prices():\n",
+    "    # Function for creating line chart showing Google stock prices over time \n",
+    "    fig = go.Figure([go.Scatter(x = df['date'], y = df['GOOG'],\\\n",
```
Comment (eliasdabbas) [inline Dash/Dash_Deploy_app_in_Naas.ipynb:222]: It's not clear where `df` is defined, and where it gets its values from? 
Maybe there should be a line before that defining it? Unless I'm missing something?

**Review by eliasdabbas — CHANGES_REQUESTED** [review]
Thanks @jravenel !

I have very minor comments, as you'll see below. 

The minimal app is great in showing how quickly it can be deployed. 

We might consider showing the interactivity of Dash, which is arguably the main feature, with a callback function. 

For example, the `stock_prices` function can take an optional `ticker` value, and dynamically retrieve that ticker's values.

File: Dash/Dash_Deploy_app_in_Naas.ipynb (hunk 1 above)
Comment (jravenel) [reply Dash/Dash_Deploy_app_in_Naas.ipynb:222]: defined in the previous line ;)

File: Dash/Dash_Deploy_app_in_Naas.ipynb [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+   "id": "7b6cda7a-77c6-464c-bdef-cfcaebe83255",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install dash   \n",
+    "!pip install dash-html-components                                         \n",
+    "!pip install dash-core-components                                     \n",
+    "!pip install plotly"
```
[the remark this answers is not in the document]
Comment (jravenel) [reply Dash/Dash_Deploy_app_in_Naas.ipynb:90]: ok done

**Comment by jravenel** [comment]
Next on this dash template, we would like to enhance with : 
- a nav bar on top 
- a layout with proper sizing of 4 elements (in square) 
- a footer

Will reference it in a new issue.

**Pull request merged**